blocks/blocks.js

```diff
@@ -196,6 +195,20 @@ const createValidators = () => {
     }
   }
 
+  // Create a color validator
+  // This can either be a column or " "
+  const validateColor = (columnName, pattern) => {
```

Comment:
`validateColor` requires two parameters, but is called with only one on line 224.